client/src/components/ContactApplicantButton.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import {
+  Form,
+  Button,
+  Modal,
+  TextArea,
+  Icon,
+  Header,
+  Grid
+} from "semantic-ui-react";
+import { sendMessage } from "../api/messages";
+
+const ContactApplicantButton = props => {
+  const [open, setOpen] = React.useState(false);
+  const [messageSent, setMessageSent] = React.useState(null);
+  const user = JSON.parse(localStorage.getItem("user"));
+  const recruiterEmailAddress = user.email;
+  const recruiterId = user.id;
+  const [message, setMessage] = React.useState({
+    profile_public_id: `${props.profilePublicId}`,
```

Comment:
You do not need to use a `${}` here, it should be just like this: `profile_public_id: props.profilePublicId`